README.md

```diff
@@ -198,8 +198,7 @@ kinto.setHeaders({
 A Kinto server exposes some of its internal settings, information about authenticated user, the HTTP API version and the API capabilities (e.g. plugins).
 
 ```js
-client.fetchServerInfo([options])
-  .then(({data}) => ...);
+const info = await client.fetchServerInfo([options]);
```

Comment:
Should it also be `const {data} = await ...` like below?

Reply:
well, no, it was a mistake, there's no `data` attribute